tests/e2e/specs/shopper/shopper-myaccount-payment-methods-add-fail.spec.js

```diff
@@ -55,4 +55,31 @@ describe( 'Payment Methods', () => {
 			} );
 		}
 	);
+
+	it( 'to no now show error when adding card on another payment gateway', async () => {
```

Comment:
How about renaming this test case to `should not show error when adding payment method on another gateway` ?